src/routes/safe/components/Balances/SendModal/screens/SendFunds/TokenSelectField/index.tsx

```diff
@@ -28,43 +38,54 @@ const SelectedToken = ({ classes, tokenAddress, tokens }) => {
           <ListItemText
             className={classes.tokenData}
             primary={token.name}
-            secondary={`${formatAmount(token.balance)} ${token.symbol}`}
+            secondary={`${formatAmount(token.balance.toString())} ${token.symbol}`}
           />
         </>
       ) : (
-        <Paragraph color="disabled" size="md" style={{ opacity: 0.5 }} weight="light">
+        <Text color="placeHolder" size="xl">
           Select an asset*
-        </Paragraph>
+        </Text>
       )}
     </MenuItem>
   )
 }
-const SelectedTokenStyled = withStyles(selectedTokenStyles)(SelectedToken)
 
-const TokenSelectField = ({ classes, initialValue, isValid, tokens }) => (
-  <Field
-    classes={{ selectMenu: classes.selectMenu }}
-    className={isValid ? 'isValid' : 'isInvalid'}
-    component={SelectField}
-    displayEmpty
-    initialValue={initialValue}
-    name="token"
-    renderValue={(tokenAddress) => <SelectedTokenStyled tokenAddress={tokenAddress} tokens={tokens} />}
-    validate={required}
-  >
-    {tokens.map((token) => (
-      <MenuItem key={token.address} value={token.address}>
-        <ListItemIcon>
-          <Img alt={token.name} height={28} onError={setImageToPlaceholder} src={token.logoUri} />
-        </ListItemIcon>
-        <ListItemText
-          primary={token.name}
-          secondary={`${formatAmount(token.balance)} ${token.symbol}`}
-          data-testid={`select-token-${token.name}`}
-        />
-      </MenuItem>
-    ))}
-  </Field>
-)
+const useSelectStyles = makeStyles(selectStyles)
+
+interface TokenSelectFieldProps {
+  initialValue?: string
+  isValid?: boolean
+  tokens: List<Token>
+}
+
+const TokenSelectField = ({ initialValue, isValid = true, tokens }: TokenSelectFieldProps): React.ReactElement => {
+  const classes = useSelectStyles()
+
+  return (
+    <Field
```

Comment:
same here perhaps is it possible to use Select component from safe-react-components.

Reply:
I prefer not, as this field is also used for Send Funds. Maybe creating a particular issue/PR for that replacement is the way to go here.